tests/qa_automation/xfstests_device.pm

```diff
@@ -60,7 +60,7 @@ sub dev_create_partition {
     type_string "umount /home\n", 5;
 
     # scratch partition number not always equal to test_partition + 1, so add these two line. poo#31156
-    $cmd = "parted --script --machine -l 2>&1| awk -F \':\' \'{if(\$3 == \"$extendpartition_end\" && \$4~/GB/) print \$1}\'";
+    $cmd = "parted --script --machine -l 2>&1| awk -F \':\' \'{if(\$3 == \"$extendpartition_end\" && (\$4~/MB/ || \$4~/GB/)) print \$1}\'";
```

Comment:
Cannot we match MB and GB in the same regexp?

Reply:
\$4~(/MB/ || /GB/)